src/styles/functions.scss

```diff
@@ -0,0 +1,4 @@
+@function getCatalogWidth($count-cards-in-row) {
+  @return ($card-width * $count-cards-in-row) +
+    (40px * 2) + ($distance-between-cards * 3);
+}
```

Comment:
i think you should change your  3 on ($count-cards-in-row - 1) and from where you got this (40px * 2)

Reply:
agree